cli/bptest/metadata_lint.go

```diff
@@ -0,0 +1,114 @@
+package bptest
+
+import (
+	"fmt"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli/bpmetadata"
+	"google.golang.org/protobuf/encoding/protojson"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sigs.k8s.io/yaml"
+)
+
+// RunMetadataLintCommand is the entry function that will run the metadata.yml lint checks.
+func RunMetadataLintCommand() {
+	metadataFile := findMetadataFile()
+	if metadataFile == "" {
```

Comment:
I dont think this needs to be an error (at least right now), maybe we can just log info as skipped?